indico/modules/rb_new/client/js/modules/bookRoom/BookRoomModal.jsx

```diff
@@ -312,14 +313,74 @@ class BookRoomModal extends React.Component {
         };
     };
 
-    renderRelatedEventsDropdown = (disabled) => {
+    renderLink = (link) => {
+        if (link) {
+            return (
+                <span styleName="link-active">
+                    <a href={link} target="_blank" rel="noopener noreferrer">
```

Comment:
since it's an internal link there's no need for "noopener noreferrer" - those are meant a security measures when linking to external/untrusted pages

Reply:
yeah, but otherwise eslint is complaining

Reply:
seems like an appropriate case to disable the warning since it doesn't apply